tools/hooks/useDayPicker.ts

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react'; // React
+import { DateUtils, RangeModifier } from 'react-day-picker'; // React DayPicker
+
+// useDayPicker inital parameters
+type useDayPicker = {
+  from: Date | undefined;
+  to: Date | undefined;
+};
+
+// useDayPicker return type
+type useDayPickerReturnType = {
+  dateRange: RangeModifier;
+  addDayToDateRange: (day: Date) => void;
+  dateRangeString: () => string;
+};
+
+/**
+ * Custom hook for managing DayPicker state
+ * @param {useDayPicker} initialDateRange - Initial date range object with from and to properties
+ * @returns dateRange state, addDayToDateRange function, dateRangeString function to display dates (YYYY-MM-DD - YYYY-MM-DD)
+ */
+export default function useDayPicker(initalDateRange: useDayPicker): useDayPickerReturnType {
+  const [dateRange, setDateRange] = useState<RangeModifier>(initalDateRange);
+
+  function addDayToDateRange(day: Date) {
+    setDateRange(DateUtils.addDayToRange(day, dateRange));
+  }
+
+  const dateRangeString = () => {
```

Comment:
This can be a value, as it will be updated whenever the `dateRange` state gets updated. We can do `let dateRangeString` and conditionally set the value as below. Then, we don't need to call the function every time we need the `dateRangeString`